provisioner/ibm-s3fs-provisioner_test.go

```diff
@@ -603,6 +609,21 @@ func Test_Provision_APIKeyWithoutServiceInstanceIDInBucketCreation(t *testing.T)
 	}
 }
 
+func Test_Provision_PVCNamespaceNotAllowedInSecrets(t *testing.T) {
+	p := getCustomProvisioner(
```

Comment:
Replace this call with `p := getFakeClientGoProvisioner(&clientGoConfig{withAllowedNamespace: true})`